src/viewing_key.rs

```diff
@@ -8,7 +8,8 @@ use cosmwasm_std::Env;
 use crate::rand::{sha_256, Prng};
 use crate::utils::{create_hashed_password, ct_slice_compare};
 
-pub const API_KEY_LENGTH: usize = 44 + 8;
+pub const VIEWING_KEY_PREFIX: &str = "api_key_";
+pub const VIEWING_KEY_LENGTH: usize = 44 /* length of base64 encoded 32 bytes */ + VIEWING_KEY_PREFIX.len();
```

Comment:
Using `/**/` insize larger expressions causes rustfmt to just ignore the line. I prefer it if you put this comment on its own line, or at the end of this one ("44 is the length of..." etc)